model/config.go

```diff
@@ -481,6 +507,71 @@ func (store *istioConfigStore) Policy(instances []*ServiceInstance, destination
 	return out
 }
 
+// DefaultProxyConfig for individual proxies
+func DefaultProxyConfig() proxyconfig.ProxyConfig {
+	return proxyconfig.ProxyConfig{
+		ConfigPath:             "/etc/istio/proxy",
+		BinaryPath:             "/usr/local/bin/envoy",
+		ServiceCluster:         "istio-proxy",
+		DrainDuration:          ptypes.DurationProto(2 * time.Second),
+		ParentShutdownDuration: ptypes.DurationProto(3 * time.Second),
+		DiscoveryAddress:       "istio-pilot:8080",
```

Comment:
shouldn't this be istio-pilot.istio-system?

Reply:
This is to remain backwards compatible.  Ingress proxy defaults to this value, and tests don't run in `istio-system`. The short name lets the test pass until we start changing defaults.